src/extension.js

```diff
@@ -71,24 +92,43 @@ function formatter(document) {
   }
 }
 
-const supportedLanguages = [
```

Comment:
Unrelated changes?

Reply:
Since this was now being used by a helper method, I thought it'd make sense to put it to move it at the top of the file instead, before the method declarations.

Reply:
Which method declaration you mean? I didn't see any reference used before declaration, and that's totally OK at the same time. My point is that don't change unnecessary codes.

Reply:
It is used in `waitForActiveSupportedDocument()`. I moved it for organization purposes, as this is how I like to organize my constants, as is how I feel most people would organize their constants as well.

Since we seem to be on a stalemate, maybe we can ask @idahogurl what they think?

Reply:
The diff shows the original codes is right up to `waitForActiveSupportedDocument`.

Reply:
@netux I don't have a preference either way. I tend to agree with @JounQin about making the least amount of changes. So if you wouldn't mind putting them above their first reference that would be great.